### DISCUSSION TITLE
Foreign language fonts are not loading properly on Chrome browser

### DISCUSSION THREAD

**Discussion by NithinPrabhakar** [getgauge/taiko#2569] [General]
While running the scripts in headless mode windows based machine the site frontend was loading properly and even the font, after switching to AWS Linux machine fonts are not loading properly(ex: Japanese, Chinese and Korean), fonts are displayed like boxes. Any suggestions on how to resolve this issue? 

**To Reproduce**
Steps (or script) to reproduce the behaviour:
1. Goto 'https://www.esteelauder.co.kr/product/681/97386/revitalizing-supreme#'
2. gauge.screenshot()

**Expected behaviour**
Font should load properly for both website and emulator 

Expected Screenshot
![image](https://user-images.githubusercontent.com/102809471/161501026-a78ee3f2-a5b0-4980-a6ab-19f3427400d7.png)

Actual Screenshot
![image](https://user-images.githubusercontent.com/102809471/161501126-d25456cb-8fc0-42b7-9a7d-bdaaaab8e81d.png)

Versions:

Taiko: [Version: 1.2.8]
OS [Amazon Linux 2]
Gauge version: 1.1.8
Commit Hash: b1501f4
Chrome: 94.0.4606.81

Plugins
-------
html-report (4.0.12)
js (2.4.0)
screenshot (0.1.0)

**Comment by zabil** [comment]
Please try installing Japanese, Korean and Chinese fonts for linux.
https://en.wikipedia.org/wiki/Help:Multilingual_support_(East_Asian)#Debian-based_GNU/Linux
